### DISCUSSION TITLE
Websocket Streams API

### DISCUSSION THREAD

**Discussion by takai-sama** [grafana/grafana-infinity-datasource#242] [Q&A] [answered]
hello, may i ask you. is it possible to streams / get response API from websocket ? (like url wss://ws3.indodax.com/ws/ ) [from this](https://github.com/btcid/indodax-official-api-docs/blob/master/ws3-websocket.md#general-information)
there is 2 ways to monitoring API from web socket, first directly connect with web socket client (extensions)
![image](https://user-images.githubusercontent.com/77669943/151104886-e15e1c6e-0b13-4014-bd2c-a904ae4d5f29.png)
second using script with library
![image](https://user-images.githubusercontent.com/77669943/151104937-940c114d-5e3e-474e-8e30-665ca2c34c42.png)

can this plugin handle the first case? any idea to do that?

**Comment by yesoreyeram** [comment, marked as answer]
No. The plugin doesn't support web socket or streaming usecases.
